demo/SpeechToText/README.md

```diff
@@ -0,0 +1,71 @@
+<h1 align="center">
+    React Native Speech-To-Text and Text-To-Speech Demo 🗣️
+</h1>
+
+<h4 align="center">This repository provides a simple yet powerful demonstration of Speech-to-Text (STT) and Text-to-Speech (TTS) functionalities using cutting-edge technologies. These demos showcase the capabilities of converting spoken language into written text and vice versa, offering a glimpse into the world of automatic speech recognition and synthesis.</h4>
+
+<br/>
+
+## 🎥 Video Demo 
+
+### Speech to text
+
+https://github.com/saigontechnology/React-Native/assets/117077260/998d2e44-9f2f-4dca-970d-e3d17c1d2d9f
```

Comment:
Can you record by other phone so we can know it's real speech to text not just show the text.
If possible please create the UI more beauty.

Reply:
done